src/tests/api/utils/ApiTestSteps.ts

```diff
@@ -15,11 +16,19 @@ const GOV_NOTIFY_INSTANCE = axios.create({ baseURL: process.env.GOVUKNOTIFYAPI }
 
 const HARNESS_API_INSTANCE : AxiosInstance = axios.create({ baseURL: constants.DEV_IPR_TEST_HARNESS_URL });
 
+const customCredentialsProvider = {
```

Comment:
@Caitlin-cooling this is dead interesting.  We should flag to @alphagov/di-devplatform-leads that this is what it takes to utilise the aws4-axios interceptor within a codebuild context.  (I think this comment suffices)